### PR TITLE
ACS-1183 ACS 7 Stacks: PostgreSQL

### PR DESCRIPTION
**Pull request by alandavis**
Upgraded to PostgreSQL 13.1 in ACS 7.0.0

### CODE REVIEW

File: helm/alfresco-content-services/values.yaml [hunk 1]
```diff
@@ -509,7 +509,7 @@ postgresql:
   # Note: Set this to false if you use an external database.
   enabled: true
   nameOverride: postgresql-acs
-  imageTag: "11.7.0"
+  imageTag: "13.1"
   commonAnnotations:
```
Comment (alandavis) [inline helm/alfresco-content-services/values.yaml:513]: Not sure this is correct, as I would not have expected 11.7.0 as the original value.

File: helm/alfresco-content-services/values.yaml [hunk 2]
```diff
@@ -534,7 +534,7 @@ postgresql-syncservice:
   enabled: true
   replicaCount: 1
   nameOverride: postgresql-syncservice
-  imageTag: "11.7.0"
+  imageTag: "13.1"
   commonAnnotations:
```
Comment (alandavis) [inline helm/alfresco-content-services/values.yaml:538]: Not sure this is correct, as I would not have expected 11.7.0 as the original value. Community used just 11.7.

**Comment by gavincornwell** [comment]
@alandavis Do you mind if I close this one as I've done the same change (and more) in this branch: https://github.com/Alfresco/acs-deployment/tree/feature/upgrade-postgres-share

NOTE: The Helm charts use the Bitnami Postgres chart which in turn use their own Postgres images hence the slightly different format.

**Review by gavincornwell — APPROVED** [review]
LGTM.

NOTE: The Helm image tag looks slightly different as we use the Bitnami Postgres Helm chart which in turn uses their own Postgres image.

**Pull request merged**